### DISCUSSION TITLE
Ordinals Coding Club Ideas

### DISCUSSION THREAD

**Discussion by raphjaph** [ordinals/ord#3011] [Ideas]
Comment below what you want us to do on our next coding stream. For some ideas you can look at issues labeled 'good-first-issue' or 'stream'.

Collected from different issues:

- Go over testing how to inscribe with TransactionTemplate
- Explain the DB tables and the indexing
- Go over how to add documentation for newcomers formatting style etc.

**Comment by elocremarc** [comment]
Resurface these. https://github.com/ordinals/ord/issues/2677#issuecomment-1815631402
Third one we could merge the documentation PR for reinscribe.

**Comment by jdareal** [comment]
PBST's.
As Cypherpunks and Quantum cats will involve running your own node, having ord wallet, and signing a PBST, I think we could all learn about how to connect our Node with Ord and Sparrow and what we are to expect with the PBST's that both of these collection will require to sign in order to get access.

**Comment by lifofifoX** [comment]
Would love any help on these 2 issues - #3014 and #3013 🙏

**Comment by Griffin-McGuire** [comment]
I'd love to see an example case of an inscription using block height as a seed for Conways game of Life :)

**Reply by cliobitcoinbank** [reply]
I've got something even better than that. Soon/

**Reply by cliobitcoinbank** [reply]
![image](https://github.com/ordinals/ord/assets/128262796/0034b313-90cc-429f-8704-818389c9915e)
Bitcoin's blocks are teeming with life!! LFG!!

**Comment by Griffin-McGuire** [comment]
Another thing. As of late, there has been some debate on the long-term reliability of recursive inscriptions. For example, if I do a recursive inscription referencing ID X, and then the numbering system changes in a future update, my recursive inscription may in fact reference Y instead. My on-chain maxi friends view this as a further abstraction from btc l1, and thus far less reliable. This is discouraging me from using recursion as it is rn (if I understand correctly) as it may swap one inscription art for another.

**Reply by cliobitcoinbank** [reply]
There is no renumbering, dont worry.  The only numbers that can change are called "sequence numbers" and they are never exposed to the frontend under pain of insurrection.

**Comment by jdareal** [comment]
I would really like to learn about OP_CAT, what it is, how it works, how to implement it on signet.

**Comment by cliobitcoinbank** [comment]
""Apologies if this is the wrong place to ask, we are getting questions both on the ordicord discord and on the r/BitcoinOrdinals subreddit in reference to multiparent and could not find relevant documentation in the Ordinals handbook for multiparent.  I was hoping to find out if this was currently possible yet or if I was correct to tell them we are on the verge of implementing this? Thanks Raphjaph, you're the man!!""

I decided to post this here instead of https://github.com/ordinals/ord/pull/2839 until I could figure out if multiparent got merged or not and maybe we could go over multiparent at the next coding club either way for woitek of ordicord and u/Ocuit of reddit who both asked recently about this feature.  Thanks Raph

**Reply by raphjaph** [reply]
We haven't implemented it yet but plan to do so soon :)
